applications/nrf_desktop/src/modules/hid_forward.c

```diff
@@ -539,7 +543,7 @@ static bool event_handler(const struct event_header *eh)
 			cast_usb_state_event(eh);
 
 		switch (event->state) {
-		case USB_STATE_POWERED:
+		case USB_STATE_ACTIVE:
```

Comment:
please leave this one, it is correct and only sets the id of the subscriber

Reply:
discussed offline - ignore the comment

Reply:
discussed offline, we can leave it as it is